src/test/resources/templates/ehr-request.hbs

```diff
@@ -1 +1 @@
-18T17:36:38.827Z</eb:Timestamp><eb:TimeToLive>2020-12-18T23:51:38.827Z</eb:TimeToLive></eb:MessageData><eb:DuplicateElimination /></eb:MessageHeader><eb:AckRequested eb:version=\"2.0\" soap:mustUnderstand=\"1\" soap:actor=\"urn:oasis:names:tc:ebxml-msg:actor:nextMSH\" eb:signed=\"false\" /></soap:Header><soap:Body><eb:Manifest eb:version=\"2.0\" soap:mustUnderstand=\"1\"><eb:Reference xlink:href=\"cid:Content1@e-mis.com/EMISWeb/GP2GP2.2A\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><eb:Description xml:lang=\"en\">RCMR_IN010000UK05</eb:Description><hl7ebxml:Payload style=\"HL7\" encoding=\"XML\" version=\"3.0\" /></eb:Reference></eb:Manifest></soap:Body></soap:Envelope>","payload":"<RCMR_IN010000UK05 type=\"Message\" xmlns=\"urn:hl7-org:v3\"><id root=\"${{conversationId}}\" /><creationTime value=\"20201218173637\" /><versionCode code=\"V3NPfIT3.1.10\" /><interactionId root=\"2.16.840.1.113883.2.1.3.2.4.12\" extension=\"RCMR_IN010000UK05\" /><processingCode code=\"P\" /><processingModeCode code=\"T\" /><acceptAckCode code=\"NE\" /><communicationFunctionRcv type=\"CommunicationFunction\" typeCode=\"RCV\"><device type=\"Device\" classCode=\"DEV\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.2.0.107\" extension=\"200000001161\" /></device></communicationFunctionRcv><communicationFunctionSnd type=\"CommunicationFunction\" typeCode=\"SND\"><device type=\"Device\" classCode=\"DEV\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.2.0.107\" extension=\"{{asidCode}}\" /></device></communicationFunctionSnd><ControlActEvent type=\"ControlAct\" classCode=\"CACT\" moodCode=\"EVN\"><author1 type=\"Participation\" typeCode=\"AUT\"><AgentSystemSDS type=\"RoleHeir\" classCode=\"AGNT\"><agentSystemSDS type=\"Device\" classCode=\"DEV\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.2.0.107\" extension=\"200000000631\" /></agentSystemSDS></AgentSystemSDS></author1><subject type=\"ActRelationship\" typeCode=\"SUBJ\" contextConductionInd=\"false\"><EhrRequest type=\"ActHeir\" classCode=\"EXTRACT\" moodCode=\"RQO\"><id root=\"FFFB3C70-0BCC-4D9E-A441-7E9C41A897AA\" /><recordTarget type=\"Participation\" typeCode=\"RCT\"><patient type=\"Patient\" classCode=\"PAT\"><id root=\"2.16.840.1.113883.2.1.4.1\" extension=\"{{nhsNumber}}\" /></patient></recordTarget><author type=\"Participation\" typeCode=\"AUT\"><AgentOrgSDS type=\"RoleHeir\" classCode=\"AGNT\"><agentOrganizationSDS type=\"Organization\" classCode=\"ORG\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.1.10\" extension=\"{{senderOdsCode}}\" /></agentOrganizationSDS></AgentOrgSDS></author><destination type=\"Participation\" typeCode=\"DST\"><AgentOrgSDS type=\"RoleHeir\" classCode=\"AGNT\"><agentOrganizationSDS type=\"Organization\" classCode=\"ORG\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.1.10\" extension=\"{{receivingOdsCode}}\" /></agentOrganizationSDS></AgentOrgSDS></destination></EhrRequest></subject></ControlActEvent></RCMR_IN010000UK05>","attachments":[]}
```

Comment:
Just for reference, `extension=\"{{sendingOdsCode}}\"` has now been changed to `extension=\"{{senderOdsCode}}\"`.